src/_scss/pages/redirect/redirectModal.scss

```diff
@@ -0,0 +1,51 @@
+@import "all";
+
+.redirect-modal {
+    background-color: $color-white;
+    .redirect-modal__header {
+        background-color: $color-primary;
+        @include display(flex);
+        @include justify-content(flex-end);
+        .redirect-modal__close-button {
+            @include flex(0 0 auto);
+            @include button-unstyled;
+            padding: rem(10);
+            svg {
+                height: rem(16);
+                width: rem(16);
+                fill: $color-white;
+            }
+        }
+    }
+    .redirect-modal__body {
+        text-align: center;
+        padding: rem(30) rem(45);
+        .redirect-modal__title {
+            @include display(inline-flex);
```

Comment:
What is the reasoning for using `inline-flex` instead of `flex`? It seems to me that you would want the title element to be block-positioned so the subsequent body content always appears below it.

Reply:
https://www.w3.org/TR/css-flexbox-1/#flex-containers